pkg/cmd/grafana-cli/utils/command_line.go

```diff
@@ -92,6 +93,24 @@ func (c *ContextCommandLine) PluginRepoURL() string {
 	return c.String("repo")
 }
 
+func (c *ContextCommandLine) GcomToken() string {
+	if c.ConfigFile() != "" {
```

Comment:
Should there be an explicit CLI arg for this too?

Reply:
It is not necessary, but we can add it.

Reply:
I would avoid allowing it as a CLI arg as well, to avoid leaking it because the command is logged by hgrun (even if at debug-level):

```
ts=2025-01-14T16:29:24.434412175Z level=debug pluginSlug=grafana-k8s-app pluginVersion=2.11.1-0987ba8 retriesLeft=0 caller=launch_config.go:71 app=hgrun hgrun_version=0.1.514-27-g98673903f slug=ephemeral1511182198973s4kh traceID=ad67c4141b8b683086a947b4cbb860e2 msg="running command" cmd="/usr/sbin/grafana-cli --pluginUrl https://storage.googleapis.com/k8s-app-dev/builds/grafana-k8s-app-latest.zip --config /etc/grafana/grafana.ini --homepath /usr/share/grafana plugins install grafana-k8s-app 2.11.1-0987ba8
```

Reply:
All good from my side - just wanted to confirm 🫡